.github/workflows/auto-approve.yaml

```diff
@@ -0,0 +1,40 @@
+name: Auto-approve Renovate PRs
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+permissions:
+  contents: write
+  pull-requests: write
+  checks: read
+  actions: read
+
+jobs:
+  auto-approve:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    # Only target safe updates from Renovate
```

Comment:
do we want to auto merge every single renovate PR or only those bumping with patch versions?

Reply:
it's not automerge, it's auto-approve it. Renovate decides which change to which branch.